java/java.sourceui/test/unit/src/org/netbeans/api/java/source/ui/HTMLJavadocParserTest.java

```diff
@@ -34,40 +33,88 @@ public HTMLJavadocParserTest(String testName) {
       super(testName);
   }
 
+//  /**
+//   * Test of getJavadocText method ised with HTML produced by standard doclet.
+//   */
+//  public void testGetJavadocText() throws MalformedURLException {
+//    URL url = HTMLJavadocParserTest.class.getResource("HTMLJavadocParser.html");
```

Comment:
nitpick: can you comment out large blocks with:
```
/*
...
*/
```
this makes it git friendlier / reduces potential merge conflicts. We had it before in TreeShims.java which caused many PRs to conflict which could have been avoided.

Reply:
OK, done.  Got heavy handed with `CTRL+/`, but it's easier when there's Javadocs. :smile: